.github/workflows/main.yaml

```diff
@@ -1,52 +1,23 @@
 name: mockingjay
 
-on: [pull_request]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - main
 
 jobs:
-  semantic-pull-request:
+  semantic_pull_request:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/semantic_pull_request.yml@v1
 
   build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: subosito/flutter-action@v1.5.3
-
-      - name: Install Dependencies
-        run: flutter packages get
-
-      - name: Format
-        run: flutter format --set-exit-if-changed .
-
-      - name: Analyze
-        run: flutter analyze lib test example
-
-      - name: Run tests
-        run: flutter test --no-pub --coverage --test-randomize-ordering-seed random
-
-      - name: Run example project tests
```

Comment:
We might want to still run the example tests as a separate job. Wdyt?

Reply:
I have added a new workflow for the example project 👍🏻